### PR TITLE
[13.0][IMP] website: Try to remove googleplus references

### LINKED ISSUE #2886: [12.0->13.0] AttributeError: 'website' object has no attribute 'social_googleplus'
After migration to 13 once you try to run odoo13 its giving error on login page.  Its because social_googleplus field no more exist in odoo13 module.

`2021-08-05 15:17:53,200 28902 INFO odoomijardin_11 odoo.modules.registry: verifying fields for every extended model 
2021-08-05 15:17:53,858 28902 INFO odoomijardin_11 odoo.modules.loading: Modules loaded. 
2021-08-05 15:17:59,312 28902 INFO ? odoo.http: HTTP Configuring static files 
2021-08-05 15:17:59,315 28902 INFO odoomijardin_11 odoo.addons.base.models.ir_http: Generating routing map for key 1 
2021-08-05 15:18:00,403 28902 ERROR odoomijardin_11 odoo.addons.http_routing.models.ir_http: 500 Internal Server Error:

Traceback (most recent call last):
  File "/opt/odoo13/odoo/addons/base/models/qweb.py", line 333, in _compiled_fn
    return compiled(self, append, new, options, log)
  File "<template>", line 1, in template_web_frontend_layout_67
AttributeError: 'website' object has no attribute 'social_googleplus'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/odoo13/odoo/addons/base/models/ir_http.py", line 234, in _dispatch
    result = request.dispatch()
  File "/opt/odoo13/odoo/http.py", line 809, in dispatch
    r = self._call_function(**self.params)
  File "/opt/odoo13/odoo/http.py", line 350, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/opt/odoo13/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/opt/odoo13/odoo/http.py", line 342, in checked_call
    result.flatten()
  File "/opt/odoo13/odoo/http.py", line 1236, in flatten
    self.response.append(self.render())
  File "/opt/odoo13/odoo/http.py", line 1229, in render
    return env["ir.ui.view"].render_template(self.template, self.qcontext)
  File "/opt/odoo13/odoo/addons/base/models/ir_ui_view.py", line 1191, in render_template
    return self.browse(self.get_view_id(template)).render(values, engine)
  File "/opt/odoo13/addons/website/models/ir_ui_view.py", line 344, in render
    return super(View, self).render(values, engine=engine, minimal_qcontext=minimal_qcontext)
  File "/opt/odoo13/addons/web_editor/models/ir_ui_view.py", line 27, in render
    return super(IrUiView, self).render(values=values, engine=engine, minimal_qcontext=minimal_qcontext)
  File "/opt/odoo13/odoo/addons/base/models/ir_ui_view.py", line 1199, in render
    return self.env[engine].render(self.id, qcontext)
  File "/opt/odoo13/odoo/addons/base/models/ir_qweb.py", line 58, in render
    result = super(IrQWeb, self).render(id_or_xml_id, values=values, **context)
  File "/opt/odoo13/odoo/addons/base/models/qweb.py", line 260, in render
    self.compile(template, options)(self, body.append, values or {})
  File "/opt/odoo13/odoo/addons/base/models/qweb.py", line 335, in _compiled_fn
    raise e
  File "/opt/odoo13/odoo/addons/base/models/qweb.py", line 333, in _compiled_fn
    return compiled(self, append, new, options, log)
  File "<template>", line 1, in template_web_login_6
  File "/opt/odoo13/odoo/addons/base/models/qweb.py", line 335, in _compiled_fn
    raise e
  File "/opt/odoo13/odoo/addons/base/models/qweb.py", line 333, in _compiled_fn
    return compiled(self, append, new, options, log)
  File "<template>", line 1, in template_web_login_layout_12
  File "/opt/odoo13/odoo/addons/base/models/qweb.py", line 340, in _compiled_fn
    raise QWebException("Error to render compiling AST", e, path, node and etree.tostring(node[0], encoding='unicode'), name)
odoo.addons.base.models.qweb.QWebException: 'website' object has no attribute 'social_googleplus'
Traceback (most recent call last):
  File "/opt/odoo13/odoo/addons/base/models/qweb.py", line 333, in _compiled_fn
    return compiled(self, append, new, options, log)
  File "<template>", line 1, in template_web_frontend_layout_67
AttributeError: 'website' object has no attribute 'social_googleplus'

Error to render compiling AST
AttributeError: 'website' object has no attribute 'social_googleplus'
Template: web.frontend_layout
Path: /t/html/body/div/footer/div[1]/div/div[2]/h2/a[5]
Node: <a t-att-href="website.social_googleplus" t-if="website.social_googleplus" rel="publisher">
              <i class="fa fa-google-plus-square"/>
            </a>
`

### PR DESCRIPTION
**Pull request by pedrobaeza**
On v13, the field `social_googleplus` has been removed, but it was put on certain places on the website, and they remain unaltered due to the noupdate=1 flag or being a COW (created on write) view, so we directly remove that part from the view if we find the exact HTML expected code that is on the definition. This is done for avoiding side effects, and it means that if you have altered somehow that part, you will need to remove it manually.

Fixes #2886

@Tecnativa

### CODE REVIEW

File: addons/website/migrations/13.0.1.0/pre-migration.py [hunk 1]
```diff
@@ -35,6 +36,47 @@ def fill_website_rewrite_name(cr):
...
+    the website, and they remain unaltered due to the noupdate=1 flag or being
+    a COW (created on write) view, so we directly remove that part from the
+    view if we find the exact HTML expected code that is on the definition.
+    This is done for avoiding side effects, and it means that if you have altered
+    somehow that part, you will need to remove it manually.
+    """
+    for key, code in [
+        (
+            "website.footer_custom",
+            '                                <a t-if="website.social_googleplus" t-att-href="website.social_googleplus"'
```
Comment (MiquelRForgeFlow) [inline addons/website/migrations/13.0.1.0/pre-migration.py:50]:
```suggestion
            ' <a t-if="website.social_googleplus" t-att-href="website.social_googleplus"'
```
Just a question, this white space is needed?

Comment (pedrobaeza) [reply addons/website/migrations/13.0.1.0/pre-migration.py:50]: Yes, it's for letting the view cleaned, removing whole line. At the end, as it has to match perfectly, this space is in the view.

Comment (MiquelRForgeFlow) [reply addons/website/migrations/13.0.1.0/pre-migration.py:50]: If everybody has the same length of white space, then ok.

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**